lib/jazzy/source_declaration/type.rb

```diff
@@ -61,6 +61,14 @@ def extension?
         kind =~ /^source\.lang\.swift\.decl\.extension.*/
       end
 
+      def extensible?
```

Comment:
I feel like Objective-C categories aren't as *lightweight* as Swift extensions. You name your Objective-C categories. Appledoc also treats them as different than their extending type.